docs/docsite/rst/dev_guide/testing/sanity/botmeta.rst

```diff
@@ -1,3 +1,5 @@
+:orphan:
+
```

Comment:
Should we also edit the one line item in this file to say botmeta no longer part of sanity tests for core? In case anyone has this page bookmarked for whatever reason or stumbles on it somehow

Reply:
We could, although I suggest doing that in a separate PR for all of the now orphaned sanity test pages. We'll need to double-check to make sure that only truly orphaned pages are actually marked as such.